SoSe16/Bericht_SoSe16_Konstanz.tex

```diff
@@ -20,13 +20,16 @@
 \definecolor{urlred}{HTML}{660000}
 \usepackage{hyperref}
 \hypersetup{
-      colorlinks=true,
-      linkcolor=black,                              % Farbe der internen Links (u.a. Table of Contents)
-      urlcolor=urlred}                              % Farbe der url-links
+	colorlinks=true,
+	linkcolor=black,                              % Farbe der internen Links 
+	%(u.a. Table of Contents)
+	urlcolor=urlred}                              % Farbe der url-links
 \parindent 0pt                                      % Absatzeinrücken verhindern
-\parskip 12pt                                       % Absätze durch Lücke trennen
+\parskip 12pt                                       % Absätze durch Lücke 
+%trennen
```

Comment:
Hmm, nicht so ganz. Irgendwie ist in dem 1. Commit nur die PDF drin. Könntest du das ganze nochmal resetten und jeweils immer die entsprechenden Dateien adden? Welche das sind, kannst du mit `git status` anzeigen, und mit `git add` zum Commit vormerken.